frontend/src/components/app/LoginForm.js

```diff
@@ -152,6 +154,8 @@ class LoginForm extends Component {
 
     request
       .then((response) => {
+        const errorType = response.data.status === 502 ? BAD_GATEWAY_ERROR : '';
```

Comment:
why we don't have a proper error code for offline error? why empty string?

Reply:
in the redux store when the  connectionErrorType is empty string like '' it will mean that there is no error.